docs/azure-simple/README.md

```diff
@@ -688,37 +688,6 @@ module.aks-gitops.module.flux.null_resource.deploy_flux: Creation complete after
 
 Apply complete! Resources: 8 added, 0 changed, 0 destroyed.
 ```
-
```

Comment:
Did you mean to take this section out?

Reply:
Yes, it is all in the following section.  Hemavathy found the double.

Reply:
In the previous section actually.